src/crawlers/cheerio_crawler.js

```diff
@@ -130,6 +131,21 @@ const DEFAULT_AUTOSCALED_POOL_OPTIONS = {
  *   this function is therefore not supported, because it would create inconsistencies where
  *   different parts of SDK would have access to a different {@link Request} instance.
  *
+ * @property {PostResponse} [postResponseFunction]
+ * A function that executes right after the HTTP request is made to the target resource and response is returned.
+ * This function is suitable for setting custom properties of response e.g. setting headers because of response parsing.
```

Comment:
I would say `overriding` instead of `setting`. Also, could you please add an example? You can use the one from the issue, overriding the content-type header for websites that respond with invalid content-type.